src/engines/kotor/gui/widgets/kotorwidget.cpp

```diff
@@ -28,6 +28,7 @@
 #include "src/aurora/gff3file.h"
 #include "src/aurora/talkman.h"
 
+#include "src/graphics/windowman.h"
```

Comment:
Add another newline after that include, because the next group is for a different subdirectory.